src/components/Error.tsx

```
@@ -0,0 +1,26 @@
+import React from "react";
+import Link from "next/link";
+import LeftArrowIcon from "@/components/icons/LeftArrowIcon";
+
+type ErrorProps = {
+    code: number;
+    message: string;
+}
+
+export default function Error( { code, message }: ErrorProps ) {
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)